src/euphorie/client/browser/webhelpers.py

```diff
@@ -1377,8 +1379,6 @@ def survey_tree_data(self):
 
         # TODO: check if guest.
         active = section == "report"
-        # The following menu items should be active even if can_edit is False.
-        disabled = section in ("", "preparation") and is_new_session
```

Comment:
Looking at the code this was always `False` so removing these two lines is anyway good right?

Reply:
They are not removed but moved up by a few lines. The `disabled` variable is reused for multiple sections (which, granted, is not ideal in terms of readability).